calendar.html

```diff
@@ -19,7 +19,7 @@ <h2>Engage in the Community</h2>
   </div>
   <a href="http://slack.layer5.io/" style="text-decoration: none !important;">
     <h3 style="text-align:center;color:aliceblue;font-size: 1.6rem;">
-      <input style="padding-bottom:.9rem;padding-top:.9rem; margin:auto; width:80%;" type="submit" value="Join the Community">
+      <input style="padding-bottom:3rem; padding-top:0.9rem; margin:auto; width:80%;" type="submit" value="Join the Community">
     </h3>
```

Comment:
Everything looks good, but maybe if we can avoid inline styling that would be great.

Reply:
what do you think about this @warunicorn19?

Reply:
@shubham-tyagi Most files have inline styling .

Reply:
Yeah @YashKamboj is right, generally we don't want to do inline styling, but here, since it's already there we'll let it be